examples/enable-operator-and-auto-instrumentation/otel-demo/otel-demo.yaml

```diff
@@ -0,0 +1,7523 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+# This file is generated by 'make generate-kubernetes-manifests'
+---
+# Source: opentelemetry-demo/charts/grafana/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/name: grafana
+    app.kubernetes.io/instance: opentelemetry-demo
+    app.kubernetes.io/version: "10.0.3"
+  name: opentelemetry-demo-grafana
+  namespace: otel-demo
+---
+# Source: opentelemetry-demo/charts/jaeger/templates/allinone-sa.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: opentelemetry-demo-jaeger
+  labels:
+    app.kubernetes.io/name: jaeger
+    app.kubernetes.io/instance: opentelemetry-demo
+    app.kubernetes.io/version: "1.45.0"
+    app.kubernetes.io/component: all-in-one
+---
+# Source: opentelemetry-demo/charts/prometheus/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/component: server
+    app.kubernetes.io/name: prometheus
+    app.kubernetes.io/instance: opentelemetry-demo
+    app.kubernetes.io/version: v2.46.0
+    app.kubernetes.io/part-of: prometheus
+  name: opentelemetry-demo-prometheus-server
+  namespace: otel-demo
+  annotations: {}
+---
+# Source: opentelemetry-demo/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: opentelemetry-demo
+  labels:
+    opentelemetry.io/name: opentelemetry-demo
+    app.kubernetes.io/instance: opentelemetry-demo
+    app.kubernetes.io/name: opentelemetry-demo
+    app.kubernetes.io/version: "1.5.0"
+    app.kubernetes.io/part-of: opentelemetry-demo
+---
+# Source: opentelemetry-demo/charts/grafana/templates/secret.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: opentelemetry-demo-grafana
+  namespace: otel-demo
+  labels:
+    app.kubernetes.io/name: grafana
+    app.kubernetes.io/instance: opentelemetry-demo
+    app.kubernetes.io/version: "10.0.3"
+type: Opaque
+data:
+  admin-user: "YWRtaW4="
+  admin-password: "YWRtaW4="
+  ldap-toml: ""
+---
+# Source: opentelemetry-demo/charts/grafana/templates/configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: opentelemetry-demo-grafana
+  namespace: otel-demo
+  labels:
+    app.kubernetes.io/name: grafana
+    app.kubernetes.io/instance: opentelemetry-demo
+    app.kubernetes.io/version: "10.0.3"
+data:
+  grafana.ini: |
```

Comment:
can we remove grafana from the demo?

Reply:
I'll see if we can remove grafana and prometheus, should be possible.

Reply:
Removed grafana, prometheus, and jaeger from otel-demo.yaml.